app/decorators/user_decorator.rb

```diff
@@ -137,12 +136,7 @@ def delete_account_bullet_key
   private
 
   def lockout_period
-    return DEFAULT_LOCKOUT_PERIOD if lockout_period_config.blank?
```

Comment:
I happened to be looking at this code, and the lockout period is specified as 10 minutes https://pages.nist.gov/800-63-3/sp800-63b.html#5132-out-of-band-verifiers so I was wondering if it _should_ be configurable

Reply:
@mitchellhenke Are you suggesting we go the other direction and _only_ hard-code this?

Reply:
@mitchellhenke My read of that section of the spec is that it's specifying a maximum duration of the sign-in attempt, not the duration of a lockout period for rate-limiting. Am I mistaken?

Reply:
That's kind of what I was thinking, yeah. We can drop a comment with a link to the specification as well? There's a few tests that reference the config value which would probably be the most difficult part of this switch.

Reply:
oh right, thanks for pointing that out, carry on 🤦🏼‍♂️